Makefile

```diff
@@ -89,6 +89,12 @@ cri-containerd:
 cc-containerd:
 	bash integration/containerd/confidential/run_tests.sh
 
+# Run the Confidential Containers tests for kubernetes.
+cc-kubernetes:
+	export K8S_TEST_UNION="confidential/agent_image.bats"
+	bash -f .ci/install_bats.sh
+	bash integration/kubernetes/run_kubernetes_tests.sh
```

Comment:
Ah okay not to worry - we've added in the suggested line (we were unsure if export worked in bash too so makes sense)! Hopefully this will allow the variable of `K8S_TEST_UNION` to be set 😄